blocks/front-page/hero/hero.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+import Button from '@rescui/button';
+import cn from 'classnames';
+
+import { useTextStyles, createTextCn } from '@rescui/typography';
+import { ThemeProvider } from '@rescui/ui-contexts';
+
+import '@jetbrains/kotlin-web-site-ui/out/components/layout';
+
+import styles from './hero.module.css';
+
+import JBLogo from '../../../assets/jetbrains-logo.svg';
+import HeroImg from './images/hero-cover.png';
+import HeroImg2x from './images/hero-cover@2x.png';
+
+interface Props {}
+
+export const HeroSection: FC<Props> = () => {
+    const textCn = useTextStyles();
+    const darkTextCn = createTextCn('dark');
+
+    return (
+        <ThemeProvider theme={'dark'}>
+            <section className={styles.heroSection}>
+                <div className={cn('ktl-layout', 'ktl-layout--center')}>
+                    <div className={styles.grid}>
+                        <div className={styles.content}>
+                            <h1 className={cn(darkTextCn('rs-hero'), styles.heroText)}>
+                                Concise.
+                                <br /> Multiplatform.
+                                <br /> Fun.
```

Comment:
It would be great, to pass this content from the index page because it tend to change.
`
<HeroSection>
<>
Concise.<br /> 
Multiplatform. <br />
Fun.
</>
`